tests/utils/BUILD

```diff
@@ -5,4 +5,9 @@ load("@npm_bazel_typescript//:index.bzl", "ts_library")
 ts_library(
     name = "utils",
     srcs = glob(["**/*.ts"]),
+    deps = [
+        "@npm//@types/mocha",
```

Comment:
this isn't needed

Reply:
If I remove this, I get this error:
`tests/utils/fixtures.ts:12:25 - error TS2503: Cannot find namespace 'Mocha'.`

Reply:
ah, my bad, of course that's needed. i looked at the file's imports and didn't see anything, but yes 'Mocha' needs to be defined. fascinating, they must be putting it on the global object or something!